docs/process/Testing.md

```diff
@@ -0,0 +1,26 @@
+**Scenario:** You are testing a PR for an issue
+
+- Checkout the associated branch, and make sure you are on it's latest commit
+- If the issue has another issue associated with it
+  - make sure you are testing the other issue at the same time 
+    - example: a new screen for managing orders requires an additional endpoint on the backend
+  - otherwise, make sure your backend/frontend is on the latest master version
+
+- For backend issues:
+  - run any required database migrations first
+  - run `mvn clean package`
+  - If there are endpoint migrations:
+    - start the application and login
+    - obtain the auth token from the `Sessions` table in the database
+    - run the following command in a terminal:
+      - `curl -X GET -H "X-Auth-Token: <token from table>" localhost:8080/migration/<endpoint name>`
+
+- For frontend issues:
+  - run `npm install`
+  - run `bower install`
+  - run `./moveAssets.sh` in /deployment/scripts
+
+- Follow the testing steps provided by the PR author
+- If the test fails, comment [Test: Fail]
+  - provide a stacktrace and reproduction steps if possible
+- Otherwise, comment [Test: Pass], and remove the **needs test** label
```

Comment:
Somehow convey that there might be multiple issues to be test passing in a single test? Ionno, words suck for me right now

Reply:
Also, add a caveat at the end for things that a tester SHOULDN'T worry about.
-If, for some reason, the testing instructions seem to not work, thats a fail. Detail what was happening, but do not correct their testing steps.
-If the testing instructions are too vague from the point of view of the tester, that's a fail. If something is ambiguous about the test, then the tester cannot be certain of the desired behavior. Testing instructions should be descriptive, verbose, and a tad pedantic.
-If the thing tested fails and the tester cannot redily identify exactly what failed, it is NOT the duty of the tester to debug the problem. Reproduction steps are enough.
-If the testing instructions require excessive unnatural acts (i.e. in very limited cases it's well and good to have to manually edit a db table during a test, but most times this is considered unnatural,) this is a fail. Testing instruction actions should only be within the domain of what a non-technical user is capable of.

Reply:
Well if there are multiple issues that are being passed in a single test, would that not imply that the PR was taking care of multiple issues at the same time? While rare, we have had instances of this before.